_data/beyond-sidebar.yml

```diff
@@ -312,7 +312,7 @@
               id: update_coupon_codes
         - title: Customers
           entries:
-            - title: Create customer
+            - title: Create customer (deprecated)
```

Comment:
@jmewes We are not sure if adding "(deprecated)" to the endpoint in the sidebar is best practice. Do you know if we marked the sidebar entry of a deprecated endpoint like this before?
What we've seen though is adding a deprecation information before the description of the endpoint. We think that in this case it would be helpful to do this instead.

Reply:
> We are not sure if adding "(deprecated)" to the endpoint in the sidebar is best practice. Do you know if we marked the sidebar entry of a deprecated endpoint like this before?

It has been a while that I added deprecation to an endpoint. I just thought that it would be helpful to immediately notice that an endpoint is deprecated.

![Selection_025](https://user-images.githubusercontent.com/5235584/185645912-2297f82f-6d0a-4d0a-a873-c8df4a176fc1.png)

![Selection_026](https://user-images.githubusercontent.com/5235584/185645937-3875b725-9f35-4e72-bd6a-714ee0f26525.png)

Reply:
We could think about adding an icon or something like that. Let's talk about this in the next CoP where the deprecation process will be discussed anyway.